### DISCUSSION TITLE
Issue with MultiSelect.Filterable component

### DISCUSSION THREAD

**Discussion by nikhilkomakula** [carbon-design-system/carbon#7852] [Help] [answered]
Hello All,

I implemented "MultiSelect.Filterable" component but whenever I click on the dropdown and fly over any of the options, I get the below error:

<img width="829" alt="image" src="https://user-images.githubusercontent.com/44344974/108545736-302e2900-730e-11eb-84ea-f70e983040df.png">

[https://codesandbox.io/s/multi-select-filter-m7cm8?file=/src/index.js:118-713](url)

```const items = [
  {
    id: "option-1",
    label: "Option 1"
  },
  {
    id: "option-2",
    label: "Option 2"
  },
  {
    id: "option-3",
    label: "Option 3"
  },
  {
    id: "option-4",
    label: "Option 4"
  },
  {
    id: "option-5",
    label: "Option 5"
  }
];

const App = () => (
  <div className="App">
    <MultiSelect.Filterable
      id={Math.random().toString()}
      placeholder="Placeholder"
      items={items}
      itemToString={(item) => (item ? item.label : "")}
      onChange={({ selectedItems }) => {
        console.log(selectedItems);
      }}
    />
  </div>
);```

Any suggestions are much appreciated. 

Thanks in advance.

**Comment by tw15egan** [comment, marked as answer]
I believe this was just fixed in https://github.com/carbon-design-system/carbon/pull/7776 and should be released today in `7.29.0`. Hope that helps!

**Reply by nikhilkomakula** [reply]
Thanks @tw15egan  for your prompt response. Appreciate it.
